packages/tag/TagSelector.tsx

```diff
@@ -0,0 +1,88 @@
+import { h, Component, emit, prop } from 'skatejs';
+import styles from './Tag.scss';
+import { Tag } from './Tag';
+
+
+// public
+interface TagSelectorProps extends JSX.HTMLProps<HTMLElement | any> {
+  onTagChange?: Function,
+  delimiter?: string,
+  tags?: any
+}
+
+export class TagSelector extends Component<TagSelectorProps> {
+  static get is() { return 'bl-tag-selector' }
+  static get props() {
+    return {
+      tags: prop.array({
+        attribute: true
+      }),
+      delimiter: prop.string({
+        attribute: true
+      }),
+    }
+  }
+
+  tags = [];
+  delimiter = ' '; // default value is space ' '
+
+  private handleInput( event ) {
+    const lastChar = event.target.value.substr(-1);
```

Comment:
we can destructure once instead of accessing long object paths

```ts
const {value} = event.target;
const lastChar = value.substr(-1);
const newValue = value.slice(0, -1).trim();
....
```